os/board/rtl8721csm/src/component/common/file_system/ftl/ftl.c

```diff
@@ -123,6 +123,56 @@ extern bool ftl_page_erase(struct Page_T *p);
 void ftl_mapping_table_init(void);
 uint16_t read_mapping_table(uint16_t logical_addr);
 
+#define FTL_SECURE 0
+#if defined(CONFIG_AMEBAD_TRUSTZONE) && (FTL_SECURE == 1)

```

Comment:
You defined a config for secure FTL here.
User needs to change this value in this file directly whenever enabling or disabling this feature.
Could you add this config to Kconfig for user to enable or disable this via menuconfig?
Is there any reason that you define and set value 0 here?

Reply:
When FTL_SECURE is define as 1, FTL will be secure it will make use of Secure API which is in TrustZone.
But this PR did not include that, is enable Secure FTL without updating TrustZone image i will create compile error.
So i turn it off by default.